apps/admin-gui/src/app/vos/components/applications-dynamic-list/applications-dynamic-list.component.ts

```diff
@@ -110,7 +115,6 @@ export class ApplicationsDynamicListComponent implements OnInit, OnChanges, Afte
       this.group?.id ?? null,
       this.getVoId()
     );
-    this.loading$.emit(this.dataSource.loading$);
```

Comment:
The spinner does not work correctly now (on refresh, on change etc.). To fix it you need to do 2 steps (i think):
Step 1: put back this line
`this.loading$.emit(this.dataSource.loading$);`